common/reviews/api/gulp-core-build.api.ts

```diff
@@ -177,16 +177,6 @@ interface IExecutable {
 export function initialize(gulp: gulp.Gulp): void;
 
 // (undocumented)
-interface ISchemaValidatorResult {
```

Comment:
@nickpape-msft
Is this a major SemVer change?  Or is nobody really relying on this interface?